### ISSUE TITLE
Interface with Tesseract and GIMP

### ISSUE THREAD

**Issue by johndharrison** [greenore/captchaSolveR#1]
Havent done OCR for some time but Tesseract and GIMP were useful. Maybe could have some options to interact via command line with https://code.google.com/p/tesseract-ocr/ and http://www.gimp.org/downloads/.

**Comment by greenore** [comment]
Yes, that's a good idea! I started a small seperate project under «[greenore/ocR](https://github.com/greenore/ocR)». As for image manipulations, do you have any experience with GIMP on the command line level? I used to work with the «[biOps](https://github.com/matiasb/biOps)» package to read images into R and do the letter rotation and seperation there. However, the package is no longer maintained. Therefore, an alternative would be great!

**EDIT**: I just rewrote the functions in «[captchaSolveR](https://github.com/greenore/captchaSolveR)» so that they no longer depend on «[biOps](https://github.com/matiasb/biOps)» and use «[EBImage](http://www.bioconductor.org/packages/release/bioc/html/EBImage.html)» instead.
